src/main/scala/org/clulab/habitus/printer/MentionInfo.scala

```diff
@@ -1,17 +1,24 @@
 package org.clulab.habitus.printer
 
+import com.typesafe.config.ConfigFactory
+import ai.lum.common.ConfigUtils._
 import org.clulab.habitus.utils.Pairable
 import org.clulab.odin.Mention
 
-case class MentionInfo(sentenceText: String, inputFilename: String, label: String) extends Pairable
+case class MentionInfo(contextWindow: String, sentenceText: String, inputFilename: String, label: String) extends Pairable
 
 object MentionInfo {
 
+  val config = ConfigFactory.load()
+  val windowSize: Int = config[Int]("VarDatesReader.contextWindowSize")
+
   def apply(mention: Mention, inputFilename: String): MentionInfo = {
     val document = mention.document
+    val sentInterval = (math.max(mention.sentence - windowSize, 0), math.min(mention.sentence + windowSize + 1, mention.document.sentences.length + 1))
+    val contextWindow = mention.document.sentences.slice(sentInterval._1, sentInterval._2).map(_.getSentenceText).mkString(" ")
```

Comment:
If that sentInterval was not a Tuple2[Int, Int] but rather a Range, then quite a bit of math could be removed and it wouldn't be necessary to program by the numbers with _._1 and _._2, which can be difficult to understand.  Next time I have to do this myself, I will probably write something like
```scala
import org.clulab.struct.Interval

val sentences = mention.document.sentences
val mentionRange = Interval(mention.sentence).indices // for lack of a Range(Int) constructor :-(
val sentRange = Range(mentionRange.start - windowSize, mentionRange.end + windowSize).intersect(sentences.indices)
val contextWindow = sentRange.map(sentences(_).getSentenceText).mkString(" ")
```

Reply:
@kwalcock, that looks great! I especially like the intersect there. Updated.